trl/models/modeling_value_head.py

```diff
@@ -143,6 +143,7 @@ def forward(
         self,
         input_ids=None,
         past_key_values=None,
+        return_past_key_values=False,
```

Comment:
Can you put that at the end (before kwargs)?

Reply:
Done